### ISSUE TITLE
iFrame or embed the settings page instead of leaving musicbox

### ISSUE THREAD

**Issue by EHLOVader** [woutervanwijk/Mopidy-MusicBox-Webclient#72]
I may be disabling the settings and power buttons to prevent anyone from accidentally rebooting the server.

But would it be possible or helpful even to embed the settings frontend into the MusicBox interface so it would be easier to return using the sidebar?

**Comment by kingosticks** [comment]
Yep I agree it would be a bit slicker. But it should be done in such a way
that  mopidy-websettings still works and looks decent when used without
musicbox-Webclient. Although I don't know if many people make use of it as
a standalone extension, it's nice it's flexible like this.

**Comment by kingosticks** [comment]
I had a quick play with this a while ago and it needs a bit of fiddling, particularly with the different style sheets being used.   Using the same/compatible styles would be the first step. I found just embedding the settings HTML into a `div` (like how the other sections work) to be easier than integrating an iframe. But both options are valid.
